src/UglyToad.PdfPig/Content/Page.cs

```diff
@@ -171,6 +171,49 @@ public decimal GetPointSize(Letter letter)
             {
                 return letter.PointSize;
             }
+
```

Comment:
I think rather than having these exporters live on the page API it's better for them to be standalone functionality that is then documented, either in the readme or the wiki. By standalone I mean the consumer can new up an instance where required, for example:

    // Constructor provides sensible defaults, or arguments are moved to the method call itself.
    ITextExporter hOcrExporter = new HOcrExporter();

It's a balance between discoverability vs making the public API too complex and I think for these export formats people are going to want to opt-in and will already know about them, whereas for most items on the public API of `Page` and `PdfDocument` they're designed so that people can autocomplete to victory. As long as we document the existence of these exporter types and provide easy on-ramps for using them, e.g.:

    hOcrExporter.Export(PdfDocument document, IPageSegmenter segmenter = null, IWordExtractor wordExtractor = null, ...);

Then people who are seeking HOcr, Alto and PageXml formats will be able to find and use them, whereas for people who are not familiar with those formats they won't be confused on encountering them on the page API.

Reply:
okay makes sense, let's make them `public` and remove them from the `Page` 👍